static/common_ui.js

```
@@ -980,3 +1071,380 @@ window.triggerScreenshot = function() {
+        if (typeof player.pause === 'function') player.pause();
+        if (typeof player.destroy === 'function') player.destroy();
+    }
+    // 清理所有实例引用
+    if (window.aplayer) {
+        window.aplayer = null;
+    }
+    if (window.aplayerInjected && window.aplayerInjected.aplayer) {
+        window.aplayerInjected.aplayer = null;
+    }
+    // 移除音乐气泡
+    document.querySelectorAll('.music-bubble').forEach(bubble => bubble.remove());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)